api/config.py

```diff
@@ -185,6 +186,7 @@ def __init__(self):
         self.REDIS_USERNAME = get_env('REDIS_USERNAME')
         self.REDIS_PASSWORD = get_env('REDIS_PASSWORD')
         self.REDIS_DB = get_env('REDIS_DB')
+        self.REDIS_CLUSTER = get_env('REDIS_CLUSTER')
```

Comment:
Suggest change it to get_bool_env() here, so that REDIS_CLUSTER can support passing true/false. 
Also, I suggest changing REDIS_CLUSTER to REDIS_CLUSTER_ENABLED for clearer semantics.